pkg/burner/job.go

```diff
@@ -274,6 +269,28 @@ func Run(configSpec config.Spec, kubeClientProvider *config.KubeClientProvider,
 	return rc, utilerrors.NewAggregate(errs)
 }
 
+// runLifecycleHook runs a user-defined shell script for a specific lifecycle hook
+func runLifecycleHook(name string, script string, background bool, configSpec config.Spec, errs *[]error, rc *int) {
+	if script == "" {
+		return
+	}
+
+	logMsg := fmt.Sprintf("Running %s command %s", name, script)
+	if !background {
+		logMsg = fmt.Sprintf("Waiting for %s command %s to finish", name, script)
+	}
+	log.Info(logMsg)
```

Comment:
The extra variable does not add much here. If anything it adds complexity to the runtime
```suggestion
	if background {
	    log.Info("Running %s command %s", name, script)
	else {
		log.Info("Waiting for %s command %s to finish", name, script)
	}
```